greenheart/tools/eco/utilities.py

```diff
@@ -1206,51 +1211,67 @@ def add_turbines(ax, turbine_x, turbine_y, radius, color):
             xlim=[
                 round(np.min(onshorex - 100), ndigits=roundto),
                 round(
-                    np.max(onshorex + onshore_substation_x_side_length + electrolyzer_side + 200),
+                    np.max(
+                        [
+                            onshorex,
+                            onshore_substation_x_side_length,
+                            electrolyzer_side,
+                            solar_side_x,
+                        ]
+                    )
+                    * 1.8,
                     ndigits=roundto,
                 ),
             ],
             ylim=[
                 round(np.min(onshorey - 100), ndigits=roundto),
                 round(
-                    np.max(onshorey + battery_side_y + electrolyzer_side + solar_side_y + 100),
+                    np.max(onshorey + battery_side_y + electrolyzer_side + solar_side_y + 100)
+                    * 1.9,
                     ndigits=roundto,
                 ),
             ],
         )
         ax[ax_index_plant].set(aspect="equal")
+
+        roundto = -3
+        point_range_x = np.max(allpoints) - np.min(allpoints)
+        point_range_y = np.max(turbine_y) - np.min(turbine_y)
+        ax[ax_index_wind_plant].set(
+            xlim=[
+                round(np.min(allpoints) - 0.5 * point_range_x, ndigits=roundto),
+                round(np.max(allpoints) + 0.5 * point_range_x, ndigits=roundto),
+            ],
+            ylim=[
+                round(np.min(turbine_y) - 0.3 * point_range_y, ndigits=roundto),
+                round(np.max(turbine_y) + 0.3 * point_range_y, ndigits=roundto),
+            ],
+        )
+        # ax[ax_index_wind_plant].autoscale()
+        ax[ax_index_wind_plant].set(aspect="equal")
+        # ax[ax_index_wind_plant].xaxis.set_major_locator(ticker.\
+        #   MultipleLocator(np.round(point_range_x*0.5, decimals=-3)))
+        # ax[ax_index_wind_plant].yaxis.set_major_locator(ticker.\
+        #   MultipleLocator(np.round(point_range_y*0.5, device_spacing=-3)))
```

Comment:
Here and in a few spots below here, do you intend to keep these commented lines on purpose?